lessons/operator/03/README.md

```diff
@@ -1,370 +1,212 @@
-- [EXERCISE 2: install and configure `sensuctl`](#exercise-2-install-and-configure-sensuctl)
-- [EXERCISE 3: use the `sensuctl --help` command](#exercise-3-use-the-sensuctl---help-command)
-- [EXERCISE 4: inspect an event in JSON format](#exercise-4-inspect-an-event-in-json-format)
-- [EXERCISE 5: explore Sensu resources using `sensuctl`](#exercise-5-explore-sensu-resources-using-sensuctl)
-- [EXERCISE 6: create an API Key for personal use](#exercise-6-create-an-api-key-for-personal-use)
-- [EXERCISE 7: export Sensu resources using `sensuctl dump`](#exercise-7-export-sensu-resources-using-sensuctl-dump)
+# Lesson 3: Using the Sensu CLI
+- [Goals](#goals)
+- [The `sensuctl` Command Line Interface (CLI)](#the-sensuctl-command-line-interface--cli)
+   - [EXERCISE 1: Install the `sensuctl` CLI](#exercise-1-install-the-sensuctl-cli)
+- [Configuration Options](#configuration-options)
+   - [EXERCISE 2: Configure the `sensuctl` CLI](#exercise-2-configure-the-sensuctl-cli)
+- [API Keys](#api-keys)
+   - [EXERCISE 3: Create an API Key](#exercise-3-create-an-api-key)
+- [Listing and Viewing Resources](#listing-and-viewing-resources)
+  - [EXERCISE 4: List Sensu Resources](#exercise-4-list-sensu-resources)
+- [Inventory Management](#inventory-management)
+  - [EXERCISE 5: List an Inventory of Nodes](#exercise-5-list-an-inventory-of-nodes)
+- [Monitoring as Code](#monitoring-as-code)
+  - [EXERCISE 6: Exporting and Updating Resource Configurations](#exercise-6-exporting-and-updating-resource-configurations)
 - [Learn more](#learn-more)
 - [Next steps](#next-steps)
 
-## Web App
+## Goals
 
-The Sensu backend process includes a web app that provides a unified view of your observability data (events, entities, and silences) and monitoring configuration (checks, handlers, etc) with user-friendly tools to reduce alert fatigue.
+In this lesson we will introduce the `sensuctl` command-line tool.
+You will learn how to install and configure the tool, then practice performing some essential operations.
 
-### Dashboard
+This lesson is intended for operators of Sensu and assumes you have [set up a local workshop environment][setup_workshop].
 
-The Sensu web app homepage is a dashboard that provides a high level overview of the overall health of the systems under management by Sensu.
-The dashboard includes a summary of active incidents, a breakdown of incidents by severity, information about entities under management, and a high level overview of the namespaces (tenants).
-This dashboard complies with Sensu RBAC, so individual users will see summary information relevant to their access.
+## The `sensuctl` Command Line Interface (CLI)
 
-![The Sensu dashboard](img/03-01-dashboard.png)
+The `sensuctl` tool, short for _Sensu Control_, gives you full control of your Sensu pipeline from a command-line environment. 
+You can use `sensuctl` interactively in a shell, or script it as part of an automated solution.
```

Comment:
One of my goals with this re-write is to avoid the previous text's tendency towards parentheticals/digressions/i.e./e.g./notes/protips/etc.. This kind of meta text adds noise to the primary language and is only sometimes helpful for some readers. 

In this case, I think readers can infer this on their own, and maybe go into automation use cases in another part of the workshop if we want to focus on that.